lib/iron_bank/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module IronBank
-  VERSION     = "4.4.1"
+  VERSION     = "4.4.2"
```

Comment:
This gem follows [semantic versioning](https://semver.org/). As such, this change will likely be a major bump, not a patch bump.